tests/RobotFramework/tests/cumulocity/mapper_recovery/recover_and_publish_software_update_message.robot

```diff
@@ -35,6 +35,7 @@ Mapper recovers and processes output of ongoing software update request
     Device Should Have Installed Software    rolldice
 
 Recovery from corrupt entity store file
+    Skip    Persistence is not a mapper responsibility anymore
```

Comment:
Don't we need to re-write this test case for `tedge-agent`?

Reply:
Yes. Skipping was just a temporary measure which got forgotten after a while.

Reply:
Resolved by d750fe